connect/transforms/src/main/java/org/apache/kafka/connect/transforms/InsertField.java

```diff
@@ -16,6 +16,15 @@
  */
 package org.apache.kafka.connect.transforms;
 
+import static org.apache.kafka.connect.transforms.util.Requirements.requireMap;
+import static org.apache.kafka.connect.transforms.util.Requirements.requireSinkRecord;
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Nit: let's avoid moving all of these imports around. Our convention is to place the `java` and static imports at the end, and moving them unnecessarily just complicates maintenance.

Reply:
Reverted these.